src/console.rs

```diff
@@ -1,21 +1,28 @@
 use std::io;
 use std::mem;
 
-use winapi::shared::minwindef::WORD;
-use winapi::um::consoleapi::{GetConsoleMode, SetConsoleMode};
-use winapi::um::wincon::{
-    self, GetConsoleScreenBufferInfo, SetConsoleTextAttribute,
-    CONSOLE_SCREEN_BUFFER_INFO, FOREGROUND_BLUE as FG_BLUE,
-    FOREGROUND_GREEN as FG_GREEN, FOREGROUND_INTENSITY as FG_INTENSITY,
-    FOREGROUND_RED as FG_RED,
+use windows_sys::Win32::Foundation::HANDLE;
+use windows_sys::Win32::System::Console::{GetConsoleMode, SetConsoleMode};
+use windows_sys::Win32::System::Console::{
+    GetConsoleScreenBufferInfo, SetConsoleTextAttribute,
+    CONSOLE_SCREEN_BUFFER_INFO, ENABLE_VIRTUAL_TERMINAL_PROCESSING,
+    FOREGROUND_BLUE, FOREGROUND_GREEN, FOREGROUND_INTENSITY, FOREGROUND_RED,
 };
 
 use crate::{AsHandleRef, HandleRef};
 
-const FG_CYAN: WORD = FG_BLUE | FG_GREEN;
-const FG_MAGENTA: WORD = FG_BLUE | FG_RED;
-const FG_YELLOW: WORD = FG_GREEN | FG_RED;
-const FG_WHITE: WORD = FG_BLUE | FG_GREEN | FG_RED;
+// TODO: windows-sys declares these types with `u32` but we need them to be
+// `u16`, so we cast them ourselves. Filed upstream:
+// <https://github.com/microsoft/windows-rs/issues/1824>
+const FG_BLUE: u16 = FOREGROUND_BLUE as u16;
+const FG_GREEN: u16 = FOREGROUND_GREEN as u16;
+const FG_INTENSITY: u16 = FOREGROUND_INTENSITY as u16;
+const FG_RED: u16 = FOREGROUND_RED as u16;
```

Comment:
These constants are already `u16` since `windows-sys 0.42` and newer. So you can simplify this.

Reply:
Fixed, thanks!